### PR TITLE
Subscribing RE to 0MQ Publisher

### PR DESCRIPTION
**Pull request by dmgav**
A built-in option to subscribe Run Engine to `bluesky.callbacks.zmq.Publisher` is added. The address of 0MQ proxy is passed with `start-re-manager` CLI parameter `--zmq-data-proxy-addr`. Identically to subscription to Kafka, the option works for Run Engine created by RE Manager and in the startup script (`start-re-manager` called with `--keep-re`). If the parameter `--zmq-data-proxy-addr` is not used, RE is not subscribed to the Publisher callback.

### CODE REVIEW

File: bluesky_queueserver/manager/start_manager.py [hunk 1]
```diff
@@ -264,6 +265,19 @@ def start_manager():
         help="Bootstrap server to connect to.",
         default="127.0.0.1:9092",
     )
+
+    parser.add_argument(
+        "--zmq-data-proxy-addr",
+        dest="zmq_data_proxy_addr",
+        type=str,
+        help="The address of ZMQ proxy used to publish data. If the parameter is specified, RE is "
+        "be subscribed to 'bluesky.callbacks.zmq.Publisher' and documents are published via 0MQ proxy. "
```
Comment (danielballan) [inline bluesky_queueserver/manager/start_manager.py:274]: The should say "RE is to be" or "RE is" or "RE will be". Not sure which one you meant.

File: bluesky_queueserver/manager/start_manager.py [hunk 2]
```diff
@@ -264,6 +265,19 @@ def start_manager():
...
+        "0MQ Proxy (see Bluesky 0MQ documentation) should be started before plans could be executed. "
```
Comment (danielballan) [inline bluesky_queueserver/manager/start_manager.py:275]:
```suggestion
        "0MQ Proxy (see Bluesky 0MQ documentation) should be started before plans are executed. "
```

**Review by danielballan — APPROVED** [review]
I have some wording suggestions to consider. Otherwise, please merge at will.

File: bluesky_queueserver/manager/start_manager.py (hunk 1 above)
Comment (dmgav) [reply bluesky_queueserver/manager/start_manager.py]: I meant 'is'.

Comment (dmgav) [reply bluesky_queueserver/manager/start_manager.py]: Fixed.

**Review by mrakitin — COMMENTED** [review]
Late to the party, but looks good to me!

**Pull request merged**